RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
+1.55.0
```

Comment:
Oh sorry that I forgot to mention that we should keep a `Unreleased` section for future changes 😄 .